### ISSUE TITLE
Some shortcuts should be reconsidered factoring in their logical consistency

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#6929]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.959
    OS Version      : Windows and Linux
    R Version       : any

### Steps to reproduce the problem

1. Move focus to terminal by pressing `"Alt+Shift+T"` on Windows or Linux.
2. For this time, try focusing in "Tools" menu by pressing `"Alt+Shift+S"`.
3. Last but not leasst, try focusing in "Session" menu by pressing `"Ctrl+Alt+Shift+S"`.
4. Compare between three keys above to see if they logically make sense, intuitive, and consistent.

### Describe the problem in detail

On Windows and Linux, `"Alt+Shift+"` key combination have been assigned towards menubar access (e.g., `"Alt+Shift+F"` for File; `"Alt+Shift+H"` for Help, etc.).

It is tempting for users to press `"Alt+Shift+T"` for "Tools" menu; `"Alt+Shift+S"` for "Session" menu.

However, `"Alt+Shift+T"` has been preserved for "Move Focus to Terminal" which is a little bit inconsistent and confusing.

### Describe the behavior you expected

Please consider reassigning `"Alt+Shift+T"` for "Tools"; `"Alt+Shift+S"` for "Session" menu on Windows and Linux.

As per "Move Focus to Terminal," we need to come up with a new shortcut key easy to press. What about ``"Alt+`"`` to move between terminal input and output area back and forth? The terminal output area has not yet been focusable or navigable; however, it's been suggested via [#6926](https://github.com/rstudio/rstudio/issues/6926).

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by kfeinauer** [comment]
Verified fixed on RSP 1.4.940-1
